content/rdi/quickstart/write-behind-guide.md

```diff
@@ -0,0 +1,224 @@
+
+- The `exclude-commands` attribute specifies which commands not to act on. For example, if you listen on key pattern that has Hash values, you can exclude the `HDEL` command so no deletions of data will propagate to the downstream database. If you don't specify this attribute, RDI Write Behind will act on all relevant commands.
+
+- The `row_format` attribute can be used with the `full` value in order to receive both the `before` and `after` sections of the payload. Note that for Write Behind events the `before` value of the key is never provided.
+
+  > Note: RDI Write Behind does not support the [`Expired`](https://redis.io/docs/manual/keyspace-notifications/#events-generated-by-different-commands) event. Therefore, keys that are expired in Redis will not be deleted from the target database automatically.
+
+### The Output Section
+
+The `output` section is critical. It specifies a reference to a connection from the `config.yaml` `connections` section:
+
+- The `uses` attribute specifies the type of **writer** RDI Write Behind will use to prepare and write the data to the target.
+  In this example, it is `relational.write`, a writer that translates the data into a SQL statement with the specific dialect of the downstream relational database.
+  For a full list of supported writers, look [here](reference/data-transformation-block-types).
+
+- The `schema` attribute specifies the schema/db to use (different database have different name for schema in the object hierarchy).
+
+- The `table` attribute specifies the downstream table to use.
+
+- The `keys` section specifies the field(s) in the table that are the unique constraints in that table.
+
+- The `mapping` section is optional and used to map a subset of the fields in the Redis Hash or JSON document to the columns that will be written to the target table.
+
+### Apply Filters & Transformations to Write Behind
+
+The RDI Write Behind jobs can apply filters and transformations to the data before it is written to the target. Specify the filters and transformations under the `transform` section.
+
+#### Filters
+
+Use filters to skip some of the data and not apply it to target.
+Filters can apply simple or complex expressions that take as arguments, the Redis entry key, fields and even the change op code (create, delete, update, etc.).
+Read [here](reference/data-transformation-block-types/filter.md) for more information about filters.
+
+#### Transformations
+
+Transformations manipulate the data in one of the following ways:
+
+- Renaming a field
+- Adding a field
+- Removing a field
+- Mapping source fields to use in output
+
+To learn more about transformations, read [here](data-transformation/data-transformation-pipeline.md).
+
+## How to Provide Target's Secrets
+
+Target's secrets (e.g. TLS certificate) can be read from a path on the Redis Nodes file system. This allows to consume secrets injected from secret stores.
```

Comment:
It felt like there was a missing word in the second sentence. I just left a possible alternative suggestion to consider. Also, I wonder if "on the Redis Nodes file system" should be changed slightly to something like: "on the Redis node's file system."
```suggestion
The target's secrets (such as TLS certificates) can be read from a path on the Redis Nodes file system. This allows the consumption of secrets injected from secret stores.
```